Testing the build and running the demo

Hello,

I wanted to try this implementation to evaluate how it would be a practical use of ShenScript. And I wanted to try the demo advertised on the main page, but it seems to fail:

```
$ brew install webpack
$ npm start
> ShenScript@0.4.0 start /Users/vresko/projects/ShenScript
> webpack-cli --watch


webpack is watching the files…

Hash: 70a761fd3a6c065ce219
Version: webpack 4.39.2
Time: 194ms
Built at: 17.08.2019 09:15:55
    Asset      Size  Chunks             Chunk Names
bundle.js  72.8 KiB    main  [emitted]  main
Entrypoint main = bundle.js
[../../../../usr/local/Cellar/webpack/4.39.2/libexec/lib/node_modules/webpack-cli/node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {main} [built]
[./dist sync recursive ^\.\/kernel\..*$] ./dist sync ^\.\/kernel\..*$ 160 bytes {main} [built]
[./index.js] 423 bytes {main} [built]
[./lib sync recursive ^\.\/config\..*$] ./lib sync ^\.\/config\..*$ 252 bytes {main} [built]
[./lib sync recursive ^\.\/frontend\..*$] ./lib sync ^\.\/frontend\..*$ 262 bytes {main} [built]
[./lib/ast.js] 6.24 KiB {main} [built]
[./lib/backend.js] 25.4 KiB {main} [built]
[./lib/config.js] 458 bytes {main} [built]
[./lib/config.node.js] 325 bytes {main} [optional] [built]
[./lib/config.web.js] 1.28 KiB {main} [optional] [built]
[./lib/frontend.js] 15.4 KiB {main} [built]
[./lib/frontend.node.js] 563 bytes {main} [optional] [built]
[./lib/frontend.web.js] 814 bytes {main} [optional] [built]
[./lib/utils.js] 1.41 KiB {main} [built]
[./shen.js] 651 bytes {main} [optional] [built]
    + 2 hidden modules

ERROR in ./lib/ast.js
Module not found: Error: Can't resolve 'astring' in '/Users/vresko/projects/ShenScript/lib'
 @ ./lib/ast.js 1:21-39
 @ ./lib/frontend.js
 @ ./lib sync ^\.\/frontend\..*$
 @ ./shen.js
 @ ./index.js
```

Unfortunately I am not that skilled with shen to debug that just myself. Any assistance to get to a demo page with an example is running would be great.

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ Still in progress:
 
 ## Prerequisites
 
-Requires recent version (10+) of [Node.js and npm](https://nodejs.org/en/download/) as well as [webpack-cli](https://www.npmjs.com/package/webpack-cli) locally installed.
+Requires recent version (10+) of [Node.js and npm](https://nodejs.org/en/download/) as well as [webpack-cli](https://www.npmjs.com/package/webpack-cli) locally installed and [http-server](https://www.npmjs.com/package/http-server) if you want to run the demo.
 
 ## Building and Testing
 
@@ -39,7 +39,7 @@ Run the following commands in order to host a simple demo page:
 npm install
 npm start
 # You should serve the root directory
-python -m SimpleHTTPServer
+http-server -p 8000
 ```
 
 If you open a browser on [localhost:8000](localhost:8000) a basic webpage will load and after a while it will display 

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ Still in progress:
 
 ## Prerequisites
 
-Requires recent version (10+) of [Node.js and npm](https://nodejs.org/en/download/).
+Requires recent version (10+) of [Node.js and npm](https://nodejs.org/en/download/) as well as [webpack-cli](https://www.npmjs.com/package/webpack-cli) locally installed.
 
 ## Building and Testing
 
@@ -33,4 +33,24 @@ Refer to [`.travis.yml`](.travis.yml) for typical build/test process. `test-*` c
 
 ## Running
 
-Run `npm start` to host a simple demo page.
+Run the following commands in order to host a simple demo page:
+
+```bash
+npm install
+npm start
+# You should serve the root directory
+python -m SimpleHTTPServer
+```
+
+If you open a browser on [localhost:8000](localhost:8000) a basic webpage will load and after a while it will display 
+the time required to load the shen environment in miliseconds (see the performance section above).
+
+If you open the javascript console at the developer tools it is possible to access to the shen global object and
+execute commands:
+
+```javascript
+shen.exec("(+ 1 1)").
+then(console.log)
+```
+
+For more information refer to the [documentation](https://shenscript.readthedocs.io/en/latest/interop.html).

